integration_tests/test_upgrade.py

```diff
@@ -126,3 +133,16 @@ def test_cosmovisor_upgrade(custom_cronos: Cronos):
 
     # test migrate keystore
     cli.migrate_keystore()
+
+    # check basic tx works
+    wait_for_port(ports.evmrpc_port(custom_cronos.base_port(0)))
+    receipt = send_transaction(
+        custom_cronos.w3,
+        {
+            "to": ADDRS["community"],
+            "value": 1000,
+            "maxFeePerGas": 1000000000000,
+            "maxPriorityFeePerGas": 10000,
+        },
+    )
+    assert receipt.status == 1
```

Comment:
maybe one good test would be do send again another upgrade proposal and check that `rsp["code"] == 0` so that we know that it will be possible to "fix" the chain after the upgrade

Reply:
seems we wont get receipt but panic if fail to remove removed storeKey